### PR TITLE
[PM-2331] Tweak renovate subject

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Tweak renovate titles to be `[deps] Team: Update <dep> to <version>`.
Disable renovate PRs for node, since it's coupled to our electron version currently.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -250,8 +260,9 @@
         "proper-lockfile"
       ],
       "description": "Vault owned dependencies",
+      "commitMessagePrefix": "[deps] Vault:",
       "reviewers": ["team:team-vault-dev"]
     }
   ],
-  "ignoreDeps": ["@types/koa-bodyparser", "bootstrap", "node-ipc", "npm", "regedit"]
+  "ignoreDeps": ["@types/koa-bodyparser", "bootstrap", "node-ipc", "node", "npm", "regedit"]
```
Comment (withinfocus) [inline .github/renovate.json:267]: ㊙️ Snuck in ...

File: .github/renovate.json [hunk 2]
```diff
@@ -192,7 +200,8 @@
         "lint-staged",
         "ts-jest"
       ],
-      "description": "SM owned dependencies",
+      "description": "Secrets Manager owned dependencies",
```
Comment (withinfocus) [inline .github/renovate.json:203]: 📓 Will want to catch this for `server` too.

**Review by withinfocus — APPROVED** [review]

File: .github/renovate.json (hunk 2 above)
Comment (Hinton) [reply .github/renovate.json:203]: Yep.

File: .github/renovate.json (hunk 1 above)
Comment (Hinton) [reply .github/renovate.json:267]: It's mentioned in the PR description ;).

**Pull request merged**